docs_src/design/adr/template.md

```diff
@@ -0,0 +1,70 @@
+# Architecture Decision Record (ADR) template <!-- Replace with ADR title -->
+This is a template for EdgeX ADR.
+The description of each section is included as comments and visible only when viewing the [source](https://raw.githubusercontent.com/edgexfoundry/edgex-docs/main/docs_src/design/adr/template.md).
+Remove this paragraph before submitting the ADR.
+
+### Submitters
+<!-- List ADR submitters
+
+Format:
+- Name (Organization)
+-->
+
+## Change Log
+<!-- List the changes to the document, incl. state, date, and PR URL.

```

Comment:
This this really practical?  Make a PR to change the document, then a PR to refer to the PR that makes the change to the document?  Isn't this what git blame is for?